modules/experimental-layers/src/bitmap-layer/README.md

```diff
@@ -0,0 +1,64 @@
+# BitmapLayer
+
+The BitmapLayer renders a bitmap at specified boundaries.
+
+```js
+import {BitmapLayer} from '@deck.gl/experimental-layers';
+
+const App = ({data, viewport}) => {
+
+  const layer = new BitmapLayer({
+    id: 'bitmap-layer',
+    bitmap: 'https://docs.mapbox.com/mapbox-gl-js/assets/radar.gif',
+    bitmapBounds: [
+      [-71.516, 37.936],
+      [-80.425, 37.936],
+      [-80.425, 46.437],
+      [-71.516, 46.437]
+    ],
+    desaturate: 0,
+    transparentColor: [0, 0, 0, 0],
+    tintColor: [255, 255, 255]
+  });
+
+  return (<DeckGL {...viewport} layers={[layer]} />);
+}
+```
+
+## Properties
+
+### Data
+
+##### `bitmap` (String|Texture2D|Image|HTMLCanvasElement)
+
+- Default `null`.
+
+##### `bitmapBounds` (Array)
+
+Supported formats:
+- Coordinates of the bounding box of the bitmap `[minX, minY, maxX, maxY]`
```

Comment:
This will mean different things in different projection systems. `minY` is at the bottom when using `MapView` and at the top when using `OrthographicView`.

The important information is that `minX, minY` is mapped to texture coords `0, 0`. So calling it `left, bottom` is clearer to the user.